About Me section on profile

stefa-n opened the pull request:
Added About Me to user profile

xusheng6 commented:
@stefa-n had a quick look at it -- please remove all changes that are solely formatting changes

stefa-n committed:
added about me functionality

stefa-n commented:
Should be correct now

stefa-n committed:
render modal conditionally

stefa-n committed:
other copilot fixes

conditional rendering of the modal
consistent capitalization
remove redundant if statement
